README.rst

```diff
@@ -32,128 +32,128 @@ Configuration
 * ``SNOW_ASSIGNMENT_GROUP`` (Optional) - The group to which the tickets should be assigned.
   If this is not provided, each call to create the tickets should be provided with an `assignment_group` argument.
   See the API documentation for more details
-* ``SNOW_DEFAULT_CHANGE_TYPE`` (Optional) - Default Change Request Type. If not provided,
+* ``SNOW_DEFAULT_CHANGE_TYPE`` (Optional) - Default Change Order Type. If not provided,
```

Comment:
What is the rationale behind this change? From what I can see, ServiceNow uses the term **Change Request** by default. This is also reinforced by the fact that the [table name](https://github.com/godaddy/django-snow/pull/7/files#diff-8a197524a23a75e0c2f4069074b656b5R23) within the code is still referred to as `change_request`